samples/Starlark/sample.star

```diff
@@ -0,0 +1,7 @@
+# This is a sample Starlark file.
```

Comment:
We prefer sample files from "real" application as these are used to train the Bayesian classifier.